scripts/ci/in_container/entrypoint_ci.sh

```diff
@@ -0,0 +1,233 @@
+    echo
+    npm run prod
+    echo
+    echo
+    popd
+fi
+
+if [[ -z "${HADOOP_HOME:=}" ]]; then
+    echo
+    echo "HADOOP_HOME not set - abort" >&2
+    echo
+    exit 1
+fi
+
+export HADOOP_HOME
+
+if [[ -z "${AIRFLOW__CORE__SQL_ALCHEMY_CONN:=}" ]]; then
+    echo
+    echo "AIRFLOW__CORE__SQL_ALCHEMY_CONN not set - abort" >&2
+    echo
+    exit 2
+fi
+
+echo
+echo "Using ${HADOOP_DISTRO:=} distribution of Hadoop from ${HADOOP_HOME}"
+echo
+
+AIRFLOW_ROOT="$(cd ${MY_DIR}; cd ../../..; pwd)"
+
+export AIRFLOW__CORE__DAGS_FOLDER="${AIRFLOW_ROOT}/tests/dags"
+
+# add test/test_utils to PYTHONPATH (TODO: Do we need it?)
+export PYTHONPATH=${PYTHONPATH:-${AIRFLOW_ROOT}/tests/test_utils}
+
+echo
+echo Backend connection: ${AIRFLOW__CORE__SQL_ALCHEMY_CONN}
+echo
+
+export AIRFLOW_HOME=${AIRFLOW_HOME:=${HOME}}
+
+echo
+echo Airflow home: ${AIRFLOW_HOME}
+echo
+
+export AIRFLOW__CORE__UNIT_TEST_MODE=True
+export HADOOP_DISTRO
+
+echo "#######################################################################"
+echo
+echo "  You can drop into ipdb debugger by adding this line to your code:"
+echo
+echo "         import ipdb; ipdb.set_trace()"
+echo
+echo "  Then run your tests with 'run-tests <TEST> --nocapture' "
+echo
+echo "#######################################################################"
+
+
+# Fix codecov build path
+# TODO: Check this - this should be made travis-independent
+if [[ ! -h /home/travis/build/apache/airflow ]]; then
+  sudo mkdir -p /home/travis/build/apache
+  sudo ln -s ${AIRFLOW_ROOT} /home/travis/build/apache/airflow
+fi
+
+# Fix file permissions
+if [[ -d $HOME/.minikube ]]; then
+    sudo chown -R airflow.airflow $HOME/.kube $HOME/.minikube
+fi
+
+if [[ ${PYTHON_VERSION} == 3* ]]; then
+    PIP=pip3
+else
+    PIP=pip2
+fi
+
+if [[ "${ENV}" == "docker" ]]; then
+    # Start MiniCluster
+    java -cp "/tmp/minicluster-1.1-SNAPSHOT/*" com.ing.minicluster.MiniCluster >/dev/null 2>&1 &
+
+    # Set up ssh keys
+    echo 'yes' | ssh-keygen -t rsa -C your_email@youremail.com -P '' -f ~/.ssh/id_rsa >/dev/null 2>&1
+    cat ~/.ssh/id_rsa.pub >> ~/.ssh/authorized_keys
+    ln -s -f ~/.ssh/authorized_keys ~/.ssh/authorized_keys2
+    chmod 600 ~/.ssh/*
+
+    # SSH Service
+    sudo service ssh restart >/dev/null 2>&1
+fi
+
+
+# Setting up kerberos
+
+FQDN=`hostname`
+ADMIN="admin"
+PASS="airflow"
+KRB5_KTNAME=/etc/airflow.keytab
+
+echo
+echo "Hosts:"
+echo
+cat /etc/hosts
+echo
+echo "Hostname: ${FQDN}"
+echo
+
+sudo cp ${MY_DIR}/krb5/krb5.conf /etc/krb5.conf
+
+echo -e "${PASS}\n${PASS}" | \
+    sudo kadmin -p ${ADMIN}/admin -w ${PASS} -q "addprinc -randkey airflow/${FQDN}"
+sudo kadmin -p ${ADMIN}/admin -w ${PASS} -q "ktadd -k ${KRB5_KTNAME} airflow"
+sudo kadmin -p ${ADMIN}/admin -w ${PASS} -q "ktadd -k ${KRB5_KTNAME} airflow/${FQDN}"
+sudo chmod 0644 ${KRB5_KTNAME}
+
+
+# Exporting XUNIT_FILE so that we can see summary of failed tests
+# at the end of the log
+export XUNIT_FILE=${AIRFLOW_ROOT}/logs/all_tests.xml
```

Comment:
What does this look like in travis? Or is this for local workflows so we have a single file to see?

Reply:
NM, I see

Reply:
Yeah. The problem is that in Travis you can't easily visualise unit test xml - unlike Jenkins/CircleCI etc. That's why I had to write the short python snippet :)